cmake/platform/Linux.cmake

```diff
@@ -11,6 +11,13 @@ if (NOT DEFINED FPRIME_USE_BAREMETAL_SCHEDULER)
 endif()
 choose_fprime_implementation(Os/File Os/File/Posix)
 choose_fprime_implementation(Os/Task Os/Task/Posix)
+choose_fprime_implementation(Os/Mutex Os/Mutex/Posix)
```

Comment:
`/` kind of remind of a file path, but in this case it should maybe correspond to the actual path, which is e.g. Os/Posix/Task and not Os/Task/Posix
We may want to either (a) rename all those targets to match the path in which they are defined e.g. Os_Mutex_Posix becomes Os_Posix_Mutex, or (b) use the actual name Os_Mutex_Posix
Thoughts?

Reply:
_ and / are the same in our system.  We probably should use _, but it is not blocking.